tests/unit/mixins/sortable-item-test.js

```diff
@@ -104,7 +104,6 @@ test('registers itself with group', function(assert) {
 
 test('deregisters itself when removed', function(assert) {
   subject.remove();
```

Comment:
You know, looking at this now, I think you were completely right to wrap this in a `run`. I’m surprised this test was ever working! Could you reinstate the `run`? Sorry!